qabelbox/src/main/java/de/qabel/qabelbox/fragments/ContactFragment.java

```diff
@@ -161,6 +164,28 @@ public void exportContact(Contact contact) {
 
     }
 
+    private String createContactFilename(String contactAlias) {
+        StringBuilder filenameBuilder = new StringBuilder();
+        filenameBuilder.append(QabelSchema.FILE_PREFIX_CONTACT);
+        filenameBuilder.append(contactAlias);
+        filenameBuilder.append(".");
+        filenameBuilder.append(QabelSchema.FILE_SUFFIX_CONTACT);
+        return filenameBuilder.toString();
```

Comment:
No need to use an explizit StringBuilder here. Concatenating the strings results in exactly the same bytecode anyway and is more readable.